### PR TITLE
Modules: Update module lifecycle and add links

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft

### CODE REVIEW

**Comment by lanceleonard** [comment]
A staged preview [is available](https://docs.redis.com/staging/jira-doc-1332/modules/modules-lifecycle/).

File: content/modules/modules-lifecycle.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+linkTitle: Module lifecycle
 description:
 weight: 7
 alwaysopen: false
 categories: ["Modules"]
 ---
-You can view the Redis Enterprise Software subscription agreement [here](https://redislabs.com/company/terms-of-use#software).
-This lifecycle fully reflects our subscription agreement.
-However, for any discrepancy between the two policies, the subscription agreement prevails.
+Redis Enterprise Software follows the [Redis Enterprise lifecycle]({{< relref "/rs/administering/product-lifecycle.md" >}}).  (For complete details, see the Redis Enterprise Software [subscription agreement](https://redislabs.com/company/terms-of-use#software).)
```
Comment (rrelledge) [inline content/modules/modules-lifecycle.md:9]:
```suggestion
Redis Enterprise Software follows the [Redis Enterprise lifecycle]({{< relref "/rs/administering/product-lifecycle.md" >}}).  (For complete details, see the Redis Enterprise Software [subscription agreement](https://redis.com/software-subscription-agreement).)
```

File: content/modules/redisgears/_index.md [hunk 2]
```diff
@@ -33,3 +33,8 @@ To learn more about the RedisGears API and understand how it works under the hoo
 Redis users typically implement caching by using the look-aside pattern. However, with RedisGears, you can implement write-behind caching strategies as well.
 
 Redis publishes RedisGears recipes to support write-behind. You can learn how to use these recipes in our write-behind caching guides for [Python]({{<relref "/modules/redisgears/python/recipes/write-behind">}}) and [Java]({{<relref "/modules/redisgears/jvm/recipes/write-behind">}}).
+
+## More info
+
+- [RedisGears Introduction](https://oss.redis.com/redisgears/intro.html)
```
Comment (rrelledge) [inline content/modules/redisgears/_index.md:39]:
```suggestion
- [RedisGears introduction](https://oss.redis.com/redisgears/intro.html)
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left a couple of small suggestions.

File: content/modules/_index.md [hunk 3]
```diff
@@ -10,11 +10,12 @@ aliases: /modules/
 ---
 Redis develops several modules that extend the core Redis feature set. Some of the features these modules provide include [querying, indexing and full-text search]({{< relref "/modules/redisearch" >}}), [JSON support]({{< relref "/modules/redisjson" >}}), and [probabalistic data structures]({{< relref "/modules/redisbloom" >}}).
 
+[Redis Stack]({{< relref "/modules/redis-stack" >}}) enables you to leverage modules quickly; it enables access to multiple modules in a single Redis database.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/_index.md]: Fixed.

File: content/modules/modules-lifecycle.md (hunk 1 above)
Comment (lanceleonard) [reply content/modules/modules-lifecycle.md]: Fixed.

File: content/modules/modules-lifecycle.md [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
-- The Minor section of the version number represents quality improvements and fixes to
+- The _Minor_ section of the version number represents quality improvements and fixes to
     existing capabilities. We increment the minor number when many quality improvements
     are added to the release.
 
 ## Modules end of life schedule {#modules-endoflife-schedule}
 
 End-of-Life for a given Major version is 18 months after the formal release of
-that version or 12 months after the release of the following version, whichever comes last.
+that version or 12 months after the release of the next-subsequent (following) version, whichever comes last.
```
[the remark this answers is not in the document]
Comment (lanceleonard) [reply content/modules/modules-lifecycle.md]: Fixed.

File: content/modules/redisgears/_index.md (hunk 2 above)
Comment (lanceleonard) [reply content/modules/redisgears/_index.md]: Fixed.

**Pull request merged**